src/main/java/com/linecorp/armeria/client/http/ImmutableHttpHeaders.java

```diff
@@ -37,7 +37,7 @@
 
     @Override
     public String get(String name) {
-        return delegate.get(name);
+        return delegate.getAsString(name);
```

Comment:
I don't like the idea adding SupressWarnings. It should be either fixed to use replacement API or leave it as it is so that IDE users can know the function is using deprecated API.